ci-operator/step-registry/sandboxed-containers-operator/create-prowjob/sandboxed-containers-operator-create-prowjob-commands.sh

```diff
@@ -373,10 +404,13 @@ tests:
       AWS_REGION_OVERRIDE: ${AWS_REGION_OVERRIDE}
       CATALOG_SOURCE_IMAGE: ${CATALOG_SOURCE_IMAGE}
       CATALOG_SOURCE_NAME: ${CATALOG_SOURCE_NAME}
+      ENABLE_MUST_GATHER: ${ENABLE_MUST_GATHER}
```

Comment:
Not related to this, but it'd really be better to create a function to generate all of these rather than this very long template prone to typos.... Something like:

```bash
...
generate_job aws peer-pods
generate_job aws coco
```